lib/output.py

```diff
@@ -28,24 +27,15 @@ def _string_to_floats(string):
     return [float(num) for num in floats]
 
 
-def get_origin(svg):
+def get_origin(svg, bounding_box):
     origin_command = global_command(svg, "origin")
 
     if origin_command:
         return origin_command.point
     else:
-        # default: center of the canvas
-
-        doc_size = list(get_doc_size(svg))
-
-        # convert the size from viewbox-relative to real-world pixels
-        viewbox_transform = get_viewbox_transform(svg)
-        simpletransform.applyTransformToPoint(simpletransform.invertTransform(viewbox_transform), doc_size)
-
-        default = [doc_size[0] / 2.0, doc_size[1] / 2.0]
-        simpletransform.applyTransformToPoint(viewbox_transform, default)
-        default = Point(*default)
-
+        minx, miny, maxx, maxy = bounding_box
```

Comment:
Cool Python trick!  You can define the method like this to automatically extract those variables:

```python
def get_origin(svg, (minx, miny, maxx, maxy)):
```

Totally your choice about whether you think this is a good idea stylistically, but I thought I'd share it :)

Reply:
Thanks for the hint, I do like your proposal better!